modules/util/util.go

```diff
@@ -63,3 +64,52 @@ func Min(a, b int) int {
 func IsEmptyString(s string) bool {
 	return len(strings.TrimSpace(s)) == 0
 }
+
+// NormalizeEOL will convert Windows (CRLF) and Mac (CR) EOLs to UNIX (LF)
+func NormalizeEOL(input []byte) []byte {
+	var right, left, pos int
+	if right = bytes.IndexByte(input, '\r'); right == -1 {
+		return input
+	}
+	tmp := make([]byte, len(input))
+	length := len(input)
+	for left < length && input[left] == '\n' {
```

Comment:
Is this faster then using something like `left = bytes.LastIndexByte(input, '\n') + 1`? My current system has `gccgo` so I'll check when I have a change to install `gc` instead for a second.

Reply:
> 
> 
> Is this faster then using something like `left = bytes.LastIndexByte(input, '\n') + 1`? My current system has `gccgo` so I'll check when I have a change to install `gc` instead for a second.

@gary-kim check the test file attached in another comment; somehow I can't attach files inside code review comments.

Reply:
So that code was munching initial `\n` to prevent them from being skipped. However, I've realised it's totally unnecessary, see below...